resources/test/fixtures/pyupgrade/UP025.py

```diff
@@ -0,0 +1,20 @@
+# These should change
+x = u"Hello"
+
+u'world'
```

Comment:
What about something like

```python
x = u'Hello "World"'
```

I don't think the current approach would handle that well.

Reply:
Good catch. I updated how I handled quotations to that this will never happen again